cmd/launcher/query_windowsupdates_windows.go

```diff
@@ -24,7 +24,7 @@ func runQueryWindowsUpdates(systemMultiSlogger *multislogger.MultiSlogger, args
 	var (
 		flagset     = flag.NewFlagSet("query-windowsupdates", flag.ExitOnError)
 		flLocale    = flagset.String("locale", "_default", "search locale")
-		flTableMode = flagset.Int("table_mode", int(windowsupdatetable.UpdatesTable), fmt.Sprintf("updates table (%d); history table (%d)", windowsupdatetable.UpdatesTable, windowsupdatetable.HistoryTable))
+		flTableMode = flagset.Int("table_mode", int(windowsupdatetable.UpdatesTable), fmt.Sprintf("updates table (%d); history table (%d); offline updates (%d)", windowsupdatetable.UpdatesTable, windowsupdatetable.HistoryTable, windowsupdatetable.UpdatesOfflineTable))
```

Comment:
I'm not sure if this should be a new table. 

Possible a `where` condition on the existing table? Maybe even defaulting to offline?

Reply:
I was thinking about following a pattern similar to `kolide_softwareupdate` vs `kolide_softwareupdate_scan` -- but this is also just for initial testing! If this setting does end up helping, then I can figure out a better option for the existing table if that's what we prefer

Reply:
My inclination is to use one table. `select * from xxx where force_fetch=true` or something.

Reply:
Will update!